library-app/configuration/config.go

```diff
@@ -37,7 +43,7 @@ func (config Config) SetLogSeverity() error {
 		log.SetLevel(log.InfoLevel)
 	case "warn":
 		log.SetLevel(log.WarnLevel)
-	case "error":
+	case "db_error":
```

Comment:
Strange naming for severity?
Sire it will work, as it is used only from configuration to here, but better would be to stick to "error" or similar "error_log_level" if "error" is already used, or it is a reserved.